packages/librespot.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Librespot < Package
+  description 'Open Source Spotify connect library'
+  homepage 'https://github.com/librespot-org/librespot'
+  version '0.1.3-1'
```

Comment:
Why did you add `-1` to the package version?

Reply:
I tend to sign my scripts with `version-revision` - incase the script is revised later on, I can remove that if I that should not be done.